2024-02-19 16:08:02,355 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-19 16:08:02,356 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Configure stats pid to 19360
2024-02-19 16:08:02,356 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-19 16:08:02,356 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-19 16:08:02,356 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-19 16:08:02,356 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-19 16:08:02,357 INFO    MainThread:19360 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_160802-zs16biei\logs\debug.log
2024-02-19 16:08:02,357 INFO    MainThread:19360 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_160802-zs16biei\logs\debug-internal.log
2024-02-19 16:08:02,357 INFO    MainThread:19360 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002A847B093C0>
2024-02-19 16:08:02,357 INFO    MainThread:19360 [wandb_init.py:init():566] calling init triggers
2024-02-19 16:08:02,358 INFO    MainThread:19360 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'n_trees': 100, 'min_samples': 2, 'max_depth': 2}
2024-02-19 16:08:02,358 INFO    MainThread:19360 [wandb_init.py:init():591] re-initializing run, found existing run on stack: l8wq7q6y
2024-02-19 16:08:02,360 INFO    MainThread:19360 [wandb_run.py:_finish():1970] finishing run mc0c0z/RandomForest/l8wq7q6y
2024-02-19 16:08:02,360 INFO    MainThread:19360 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-19 16:08:02,360 INFO    MainThread:19360 [wandb_run.py:_restore():2193] restore
2024-02-19 16:08:02,360 INFO    MainThread:19360 [wandb_run.py:_restore():2199] restore done
2024-02-19 16:08:05,747 INFO    MainThread:19360 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-19 16:08:05,747 INFO    MainThread:19360 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-19 16:08:05,753 INFO    MainThread:19360 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-02-19 16:08:05,851 INFO    MainThread:19360 [wandb_init.py:init():616] starting backend
2024-02-19 16:08:05,851 INFO    MainThread:19360 [wandb_init.py:init():620] setting up manager
2024-02-19 16:08:05,853 INFO    MainThread:19360 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-02-19 16:08:05,854 INFO    MainThread:19360 [wandb_init.py:init():628] backend started and connected
2024-02-19 16:08:05,861 INFO    MainThread:19360 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-02-19 16:08:05,861 INFO    MainThread:19360 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-19 16:08:05,861 INFO    MainThread:19360 [wandb_init.py:init():720] updated telemetry
2024-02-19 16:08:05,863 INFO    MainThread:19360 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-19 16:08:07,371 INFO    MainThread:19360 [wandb_run.py:_on_init():2262] communicating current version
2024-02-19 16:08:07,553 INFO    MainThread:19360 [wandb_run.py:_on_init():2271] got version response 
2024-02-19 16:08:07,554 INFO    MainThread:19360 [wandb_init.py:init():804] starting run threads in backend
2024-02-19 16:08:07,651 INFO    MainThread:19360 [wandb_run.py:_console_start():2241] atexit reg
2024-02-19 16:08:07,651 INFO    MainThread:19360 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-02-19 16:08:07,651 INFO    MainThread:19360 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-02-19 16:08:07,651 INFO    MainThread:19360 [wandb_run.py:_redirect():2186] Redirects installed.
2024-02-19 16:08:07,652 INFO    MainThread:19360 [wandb_init.py:init():847] run started, returning control to user process
2024-02-19 16:23:57,473 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-19 16:23:57,473 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Configure stats pid to 19360
2024-02-19 16:23:57,474 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-19 16:23:57,474 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-19 16:23:57,474 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-19 16:23:57,474 INFO    MainThread:19360 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-19 16:23:57,475 INFO    MainThread:19360 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_162357-eob1xzb2\logs\debug.log
2024-02-19 16:23:57,475 INFO    MainThread:19360 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_162357-eob1xzb2\logs\debug-internal.log
2024-02-19 16:23:57,475 INFO    MainThread:19360 [wandb_init.py:init():566] calling init triggers
2024-02-19 16:23:57,475 INFO    MainThread:19360 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'n_trees': 100, 'min_samples': 2, 'max_depth': 10}
2024-02-19 16:23:57,476 INFO    MainThread:19360 [wandb_init.py:init():591] re-initializing run, found existing run on stack: zs16biei
2024-02-19 16:23:57,478 INFO    MainThread:19360 [wandb_run.py:_finish():1970] finishing run mc0c0z/RandomForest/zs16biei
2024-02-19 16:23:57,478 INFO    MainThread:19360 [jupyter.py:save_history():445] not saving jupyter history
2024-02-19 16:23:57,478 INFO    MainThread:19360 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-19 16:23:57,479 INFO    MainThread:19360 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-19 16:23:57,479 INFO    MainThread:19360 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-02-19 16:23:57,479 INFO    MainThread:19360 [wandb_run.py:_restore():2193] restore
2024-02-19 16:23:57,479 INFO    MainThread:19360 [wandb_run.py:_restore():2199] restore done
2024-02-19 16:24:05,404 INFO    MainThread:19360 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-19 16:24:05,404 INFO    MainThread:19360 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-19 16:24:05,410 INFO    MainThread:19360 [wandb_run.py:_footer_sync_info():3825] logging synced files
